src/library.js

```diff
@@ -2463,8 +2382,9 @@ LibraryManager.library = {
 
   // Represents whether emscripten_get_now is guaranteed monotonic; the Date.now
   // implementation is not :(
+  $nowIsMonotonic__internal: true,
 #if MIN_IE_VERSION <= 9 || MIN_FIREFOX_VERSION <= 14 || MIN_CHROME_VERSION <= 23 || MIN_SAFARI_VERSION <= 80400 // https://caniuse.com/#feat=high-resolution-time
```

Comment:
This conditional directive is duplicated three times in this file. As an possible follow-up, perhaps we could make a `performance.now()` polyfill in [`src/polyfill`](https://github.com/emscripten-core/emscripten/tree/main/src/polyfill) that calls `Date.now() - nowOffset` if unsupported (e.g. https://gist.github.com/paulirish/5438650)? Then we could just call that polyfill throughout this file.